addon/components/nrg-modal-container/component.js

```diff
@@ -1,17 +1,23 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
-import { alias, bool, filterBy } from '@ember/object/computed';
+import { filterBy, notEmpty, readOnly } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
 import layout from './template';
 
 export default Component.extend({
   layout,
   tagName: '',
+
   modalService: service('modal'),
-  openModals: alias('modalService.items'),
-  hasOpenModals: bool('openModals.length'),
+
+  openModals: readOnly('modalService.openModals'),
+  activeModal: readOnly('modalService.activeModal'),
+
+  hasOpenModals: readOnly('modalService.hasOpenModals'),
+
   sidebarItems: filterBy('openModals', 'sidebar', true),
-  hasSidebar: bool('sidebarItems.length'),
+
+  hasSidebar: notEmpty('sidebarItems'),
```

Comment:
Can we change this to `readOnly('activeModal.sidebar')`?
And get rid of `sidebarItems`.

This way a sidebar under another modal will not cause the sidebar dimmer to show.